test/crds.yaml

```diff
@@ -67,3 +70,55 @@ items:
       kind: GlobalNetworkPolicy
       plural: globalnetworkpolicies
       singular: globalnetworkpolicy
+- apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
this is a dupe, unless i'm misreading the entry above.

Reply:
@robbrockbank It was a duplicate. I've removed it.